components/pages/home/hero.tsx

```
@@ -0,0 +1,30 @@
+import React, { FunctionComponent } from "react";
+import styles from "@styles/components/pages/home/hero.module.css";
+
+import Typography from "../../UI/typography/typography";
+import { TEXT_TYPE } from "@constants/typography";
+import SplitTitle from "./splitTitle";
+
+
+const Hero: FunctionComponent<{}> = ({}) => (
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)